app/assets/javascripts/discourse/lib/emojis-store.js.es6

```diff
@@ -0,0 +1,50 @@
+import KeyValueStore from "discourse/lib/key-value-store";
+
+const EMOJI_USAGE = "emojiUsage";
+const EMOJI_SELECTED_DIVERSITY = "emojiSelectedDiversity";
+const TRACKED_EMOJIS = 15;
+const STORE_NAMESPACE = "discourse_emojis_";
+
+export default class EmojiStore {
```

Comment:
I notice this is `EmojiStore` and elsewhere it's EmojisStore (with an s). It looks like in English it can be either emojis or emoji but we should standardize on one. I think our codebase uses "emoji" much more for the plural format so we should probably stick with that.